plugins/generator-1.20.6/neoforge-1.20.6/templates/potioneffect.java.ftl

```diff
@@ -68,16 +68,7 @@ public class ${name}MobEffect extends <#if data.isInstant>Instantenous</#if>MobE
 
 	<#if hasProcedure(data.activeTickCondition) || hasProcedure(data.onActiveTick)>
 		@Override public boolean shouldApplyEffectTickThisTick(int duration, int amplifier) {
-			<#if hasProcedure(data.activeTickCondition)>
-				return <@procedureOBJToConditionCode data.activeTickCondition/>;
-			<#else>
-				return true;
-			</#if>
-		}
-	</#if>
-
-	<#if hasProcedure(data.onActiveTick)>
-		@Override public void applyEffectTick(LivingEntity entity, int amplifier) {

```

Comment:
applyEffectTick exists in 1.20.6, so please keep using it and call super method and store return value to return it at the end of the function

Reply:
but applyEffectTick now it is a boolean function

Reply:
>  call super method and store return value to return it at the end of the function

Reply:
Done